### PR TITLE
Remove authentication from launcher

### LINKED ISSUE #68: Inaccurate error message when creating account with invalid email
# Issue
When creating an account with an invalid email address (e. g. `adfassdsadfasdffsad`) the following error is shown:

StationHub_2020-04-05_14-07-46

This is inaccurate - the user hasn't created an account yet, and no email is sent.

The user is then returned to the login screen. They lose the progress they made in the registration form; They have to enter their user name name and password again.

# Suggested resolution
* Make the error message more generic

or
* Inform the user that they have a typo in their email address

and optionally:
* return the user to the account creation screen instead of the login screen.

### LINKED ISSUE #165: Remove the login feature from Stationhub
As a launcher, it shouldn't also handle login as that depends on the game itself. 

A future fork could be hosting their own solution for authentication, as some will need to, since our authentication system also handles character creation and persistence and they might have their own customisations.

Login and account creation should only be handled by the game itself and a website that communicates with the particular API the game build is using.

### PR DESCRIPTION
**Pull request by CorruptComputer**
Removed all the views relating to the login feature, as this will be handled by the game instead.
Replaced the profile picture on the launcher with the Unitystation logo.

image

Fixes #165
Fixes #68

### CODE REVIEW

**Comment by corp-0** [comment]
tested it locally and seems to work perfectly fine. I could Login to my account, created a completely new one and also tried to join a server with my stored credentials. The game handles all cases very well.

**Comment by CorruptComputer** [comment]
Oh awesome, those were all things I was worried about testing with this change. I'll leave this as draft until 931 is out and it can be the first one in for 932.

File: UnitystationLauncher/Views/LauncherView.xaml [hunk 1]
```diff
@@ -7,56 +7,12 @@
...
-                        <Button Margin="2" Command="{Binding #ContextMenu.Open}">
-                            <Button.Styles>
-                                <Style Selector="Button">
-                                    <Setter Property="BorderThickness" Value="0" />
+            <Border Margin="5 5 0 6">
+                <Image Source="avares://StationHub/Assets/unityico.png" Margin="0 2 1 2" />
+            </Border>
 
-                                    <Setter Property="Background">
-                                        <ImageBrush Source="/Assets/buttoncontext.png" />
```
Comment (CorruptComputer) [inline UnitystationLauncher/Views/LauncherView.xaml:25]: Check if this file is used anywhere else, and remove it if not.

File: UnitystationLauncher/Views/LauncherView.xaml [hunk 2]
```diff
@@ -7,56 +7,12 @@
...
-                                </Style>
-                            </Button.Styles>
-                            <Button.ContextMenu>
-                                <ContextMenu Name="ContextMenu" PlacementMode="Bottom">
-                                    <MenuItem Header="Refresh" Command="{Binding Refresh}" />
-                                    <MenuItem Header="Logout" Command="{Binding Logout}" />
-                                </ContextMenu>
-                            </Button.ContextMenu>
-                        </Button>
-                        <Image Source="/Assets/userico.jpg" HorizontalAlignment="Left" ToolTip.Tip="Profile Icon"
```
Comment (CorruptComputer) [inline UnitystationLauncher/Views/LauncherView.xaml:46]: Check if this file is used anywhere else, and remove it if not.

File: UnitystationLauncher/Views/LauncherView.xaml [hunk 3]
```diff
@@ -7,56 +7,12 @@
...
-                               Margin="0 2 1 2" />
-                    </StackPanel>
-                </Border>
-                <Viewbox Height="22" HorizontalAlignment="Left" VerticalAlignment="Center" ToolTip.Tip="Username">
-                    <Border Margin="4 0 0 0" VerticalAlignment="Center">
-                        <Border.Background>
-                            <ImageBrush Source="/Assets/userbg.png" Stretch="Fill" />
```
Comment (CorruptComputer) [inline UnitystationLauncher/Views/LauncherView.xaml:53]: Check if this file is used anywhere else, and remove it if not.

**Comment by corp-0** [comment]
I don't see why not include this in 931

**Comment by CorruptComputer** [comment]
Alright, I'll get those couple of things I want to check done and update the changelog.

**Comment by corp-0** [comment]
all good to merge?

**Comment by CorruptComputer** [comment]
Yeah should be good to go

**Review by corp-0 — APPROVED** [review]

**Pull request merged**